### ISSUE TITLE
[Feature Request] A way to /stop/ a macro that is currently running

### ISSUE THREAD

**Issue by frigvid** [Ryochan7/DS4Windows#2404]
As far as I can tell there is currently no way to actually stop a macro once it has been started. If you're going to have macros, one also needs a way to stop them, otherwise one might as well just skip them all together because they're useless the moment something goes wrong.

For example, a looping macro can't be stopped.

One way would be to have it stop by simply pressing the set combination again. Right now that just launches the macro _again_, which leaves you with two running at the same time, instead of toggling it from on to off and so on and so forth.

TL;DR: Please add a way to _stop_ a (or all) currently active macros.
